src/absenceDays/index.ts

```diff
@@ -1,20 +1,20 @@
 import BaseApi from '../baseApi';
-import responseHandler from '../utils/response/responseHandlers';
 import RequestParams from '../utils/requestParams/requestParams';
 import { AbsenceDay } from './types';
+import { required, requiredList } from '../utils/response/responseHandlers';
 
 export default class AbsenceDays extends BaseApi {
-  public readonly resourceName = 'absencesDays';
+  public readonly resourceName = 'absenceDays';
 
   public read(requestParams?: RequestParams<AbsenceDay> | Object): Promise<AbsenceDay[]> {
     const params = requestParams instanceof RequestParams ? requestParams.getParams() : requestParams;
     const response = this._get<AbsenceDay[]>(`${this.getResourceName()}/read`, { params });
-    return responseHandler.requiredList(response);
+    return requiredList(response);
   }
 
   public readById(id: number, requestParams?: RequestParams<AbsenceDay> | Object): Promise<AbsenceDay> {
     const params = requestParams instanceof RequestParams ? requestParams.getParams() : requestParams;
-    const response = this._get<AbsenceDay>(`${this.getResourceName()}/read/${id}`, { params });
-    return responseHandler.required(response);
+    const response = this._get<AbsenceDay[]>(`${this.getResourceName()}/read/${id}`, { params });
+    return required(response);
```

Comment:
I think I personally would've preferred the old style, with `responseHandler.required()`, since it's more explicit and having just `required()` confuses me a bit, but it's a matter of taste, I guess.

Reply:
I would also prefer that. I removed it since responseHandler was a class and needed to be instantiated before exporting. Do you think exporting it like the following, so that we can still call it with `responseHandler.something()` would make sense instead? 
`export const responseHandlers = { required, requiredList... }`

Reply:
Possibly, but I'm not sure if changes to the export are necessary, you can do something like
```
import * as responseHandlers from '../utils/response/responseHandlers';
```
(nor sure if i got the syntax exactly right though)